index.html

```diff
@@ -46,778 +46,778 @@ <h3>Spring 2023, Class: Mon, Wed 4:30pm-5:50pm, Gates B1</h3>
-							</td>
-						</tr>
-						<tr>
-							<td><b>Week 2</b><br />
-								Mon, April 10
-							</td>
-							<td><i><span class="label label-lecture text-base">Lecture</span></i> 
-								<a href="slides/cs224r-mdp-pg.pdf">MDPs and Policy Gradients</a>
-							</td>
-							<td>
-							<!--
+                            of concepts including, but not limited to (stochastic) gradient descent and
+                            cross-validation,
+                            and pre-requisites such as probability theory, multivariable calculus, and linear algebra.
+                        </p>
+                        <p><b>Some familiarity with deep learning</b>: The course will build on deep learning concepts
+                            such as
+                            backpropagation, convolutional networks, and recurrent neural networks. The assignments will
+                            involve programming in PyTorch. The first week will include a short PyTorch review tutorial.
+                        </p>
+                        <p><b>Some familiarity with reinforcement learning</b>: We will assume some familiarity with the
+                            basics
+                            of reinforcement learning. For introductory material on RL and Markov decision processes
+                            (MDPs),
+                            see CS221’s <a href="https://stanford-cs221.github.io/autumn2022/modules/">lectures</a> on
+                            MDPs and
+                            RL, or see Chapters 3 and 4 of <a
+                                href="http://incompleteideas.net/book/RLbook2020.pdf">Sutton & Barto</a>.</p>
+
+                        <h3>Lecture Slides and Videos:</h3>
+                        <p>Lecture slides will be posted on the course website at least 15 minutes before each lecture.
+                        </p>
+
+                        <h3>Previous Offerings</h3>
+                        <ul>
+                            <li><a href="spring_2023/index.html">Spring 2023</a></li>
+                        </ul>
+                </div>
+
+                <div class="hrline">
+                    <hr />
+                </div>
+                <div class="row clearfix">
+                    <h3> Staff </h3>
+                    <div class="row clearfix">
+                        <div class="col-md-2 column">
+                            <img class="img-thumbnail square-image" src="images/chelsea_finn_square_crop.jpg"
+                                alt="Chelsea Finn" width="160" height="160">
+                            <h4>Prof. Chelsea Finn</h4>
+                            <h5>Instructor</h5>
+                            <h5>
+                                </h4>
+                                <h5>OH: Wed 5 &ndash; 6pm</h5>
+                                <h5>Location: Gates 358</h5>
+                                <h5><a href="https://ai.stanford.edu/~cbfinn/">Webpage</a></h5>
```

Comment:
I think this looks a little awkward esp since it's the only webpage. Maybe remove, or just add a link to the name rather than a separate "Webpage" text.